tests/test_estimators/test_umap_reconstruction.py

```diff
@@ -5,6 +5,8 @@
 from sklego.decomposition import UMAPOutlierDetection
 from tests.conftest import general_checks, nonmeta_checks, outlier_checks, select_tests
 
+pytestmark = pytest.mark.umap
```

Comment:
I've never used this feature before, just to check my understanding ... does this set the marker on every test in this file?

Reply:
Yes correct, it _marks_ the entire module/file!